1208-projects/project_wizard.feature

```diff
@@ -0,0 +1,153 @@
+Feature: User edits Project
+# First three lines cover Why, Who, and What, following scenarios covers the How
+In order to document a project 
+As a logged in User or as a Curator
+I want to edit existing and create new projects
+
+    @xxx
+    Scenario: User sees the Project Wizard
+        Given User opens My Projects
+        When they click Create New Project Button
+        Then the User is the Project Owner
+        And they see tabs:
+            | Description  |
+            | Participants |
+            | Financing    |
+        And they see the Description tab
+        And they see fields:
+            | Title                    |
+            | Summary                  |
+            | Coordinating Institution |
+            | Start Date               |
+            | End Date                 |
+            | Internal Project Code    |
+        And they see a Support Button
+        And they see a Save Button
+        And they see a Next Button
+
+    @xxx
+    Scenario: Curator opens a Project in the Project Wizard
+        # Same end result as "User Edits a Project in the Project Wizard"
+        # but this one enables a Curator to edit projects that he's not an
+        # participant of
+        Given User opens Landing Page for Project
+        And User is Curator on Project's Project Owner's Institution
+        And the project lacks an Approval of type "REK"
+        # A "REK" Approved project is a Health Project. 
+        # See health_related_projects.feature for details.
+        When they click the Edit button
+        Then User sees the Project Wizard
+        And it contains data about the Project
+
+    @xxx
+    Scenario Outline: User opens Participants tab for Project
+        Given User sees the Project Wizard
+        And User has the "<Role>" role in the project's scope
+        When they click the Participants tab
+        Then they see lists of:
+            | Project Managers     |
+            | Project Participants |
+        And they see Buttons:
+            | Add Project Manager     |
```

Comment:
Denne skal egentlig ikke vises for `Local Project Manager`? (Ser den rollen mangler i scenario `User opens Dialog for adding Project Manager`)

Reply:
Har duplisert scvenario og slettet PM-knapp på den eine og roller på den andre.